rel/expr_tuple.go

```diff
@@ -140,6 +140,10 @@ func (e *TupleExpr) String() string { //nolint:dupl
 
 // Eval returns the subject
 func (e *TupleExpr) Eval(ctx context.Context, local Scope) (Value, error) {
+	if cmd, isStr := ctx.Value(CmdIdentifier{Name: "test"}).(string); isStr && cmd == "test" {
+		return e.evalTests(ctx, local)
+	}
+
```

Comment:
This shouldn't be required. `test.Test` should explicitly unpack the TupleExpr and evaluate each component.

Reply:
```
type TupleExpr struct {
	ExprScanner
	attrs   []AttrExpr
	attrMap map[string]Expr
}
```
This is current definition of `TupleExpr`, so it has to change `attrs` to `Attrs` to unpack in `test.Test`. Is it acceptable? Or it can add new functions in `expr_tuple.go` or package `rel` for `test`.